mise.toml

```diff
@@ -0,0 +1,16 @@
+[tools]
+
+# Core Dependencies
+go = "1.22.7"
+
+# Go Dependencies
+"go:github.com/golangci/golangci-lint/cmd/golangci-lint" = "1.64.5"
+"go:go.uber.org/mock/mockgen" = "0.5.0"
+"go:gotest.tools/gotestsum" = "1.12.0"
+"go:github.com/segmentio/golines" = "0.12.0"
```

Comment:
oh nice so mise provides a wrapper framework on top of dev dependencies that allows for steamlined version mgmt vs having to inexplicitly manage them across things like Make and CI?

Reply:
Yep exactly and the mise github action that we use in ci caches all the dependencies so there's no extra incurred download cost on every ci run.